### DISCUSSION TITLE
Here is the French translation (based upon latest en-US) for "GetMyIP_0.11.2"

### DISCUSSION THREAD

**Discussion by logounet** [Timthreetwelve/GetMyIP#105] [General]
Hello,

Here is the French translation (based upon latest en-US) for "GetMyIP_0.11.2"

See attached file

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/17292140/Strings.fr-FR.xaml.zip)

Best regards

**Comment by Timthreetwelve** [comment]
@logounet, thanks for your update. I really appreciate it. I've added a few more strings to en-US after receiving it.

``` XML
    <!-- Begin changes on 2024/10/31 @ 02:05 UTC-->
    <!-- A | External_LastRefresh -->
    <!-- A | SettingsItem_RefreshRestoredWindow -->
    <!-- A | SettingsItem_ShowRefreshTime -->
    <!-- End changes on 2024/10/31 @ 02:05 UTC-->

    <!-- Begin changes on 2024/11/04 @ 20:15 UTC-->
    <!-- A | SettingsItem_TranslationToolTipLine1 -->
    <!-- A | SettingsItem_TranslationToolTipLine2 -->
    <!-- End changes on 2024/11/04 @ 20:15 UTC--> 
```

Thanks again,
Tim

**Comment by logounet** [comment]
Hello,

Here is the French translation (based upon latest en-US)"

See attached file

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/17629794/Strings.fr-FR.xaml.zip)

Best regards

**Comment by Timthreetwelve** [comment]
Thank you!

**Comment by logounet** [comment]
Hello,

Here is the French translation for "Get My IP 0.12.1" (based upon latest en-US)"

See attached file


[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/17852078/Strings.fr-FR.xaml.zip)


Best regards

**Comment by Timthreetwelve** [comment]
Thanks, @logounet, I will have it in the next release.

Tim

**Comment by Timthreetwelve** [comment]
@logounet, it seems that I've done it again. I've made a change since receiving your latest update.

```
    <!-- Begin changes on 2024/11/22 @ 22:25 UTC -->
    <!-- D | SettingsItem_ImportWarning -->  This was deleted.
    <!-- A | MsgText_ImportSettingsRestart -->
    <!-- End changes on 2024/11/22 @ 22:25 UTC -->
```

Thanks,
Tim

**Comment by logounet** [comment]
Hi,

Ok, here is the last one

[Strings.fr-FR.xaml.zip](https://github.com/user-attachments/files/17879002/Strings.fr-FR.xaml.zip)


Cordially

**Comment by Timthreetwelve** [comment]
@logounet, thank you very much.

Tim
